code/radar/radardradis.cpp

```diff
@@ -129,8 +129,13 @@ void HudGaugeRadarDradis::plotBlip(blip* b, vec3d *pos, float *alpha)
 			fade_multi *= 2.0f;
 		}
 	}
+	
+	b->time_since_update += flFrametime;
+	if (abs(vm_vec_dot(&sweep_normal_x, pos)) < 0.01f) {
```

Comment:
This condition would benefit from a comment to explain what it does because I have no idea what this condition checks.